src/main/java/it/pagopa/pn/bff/utils/ConverterUtils.java

```diff
@@ -0,0 +1,14 @@
+package it.pagopa.pn.bff.utils;
```

Comment:
Why we don't use the same approach used in NotificationDetailMapper?

Reply:
There's no specific reason. I've converted this utility to a mapper